packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -54,16 +75,15 @@ describe('Sequence - ', () => {
         const result = await this.invoke(route, args);
         // Prepend 'MySequence' to the result of invoke to allow for
         // execution verification of this user-defined sequence
```

Comment:
We should remove this useless comment, the intent is obvious from the code below.